OpenRA.Mods.Cnc/Traits/MadTank.cs

```diff
@@ -189,9 +189,11 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 				if (!order.Queued)
 					self.CancelActivity();
 
-				self.SetTargetLine(target, Color.Red);
+				self.ShowTargetLines();
 				self.QueueActivity(new MoveAdjacentTo(self, target));
-				self.QueueActivity(new CallFunc(StartDetonationSequence));
+				var det = new CallFunc(StartDetonationSequence);
+				det.SetTargetLineNode(target, Color.Red, true);
```

Comment:
Why isn't this already handled by the `MoveAdjacentTo`?

Reply:
I don't get this question? MoveAdjacentTo is supposed to be drawn in green (or not drawn at all) so I'm painting the detonation activity red. This activity appears when you order a Mad tank to queue a move order then queue the detonation order by right-clicking on the target structure.

Reply:
Now I get what you mean, seeing the comment below. As mentioned below, MoveAdjacentTo isn't really drawn as the color of the line depends on the activity that comes after MoveAdjacentTo. (e.g., Demolition, ExternalCapture, Dock, etc.)